src/sentry/static/sentry/app/views/settings/components/forms/dateTimeField.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import InputField from './inputField';
+
+type Props = InputField['props'];
```

Comment:
We should probably omit `type` since that's overriden.

Reply:
@EvanPurkhiser, `Omit<Props, 'type'>`? So that people know not to pass `type` to these, right?

Reply:
yah